core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.component.html

```diff
@@ -249,7 +249,13 @@ <h3>Skills</h3>
   }
   .subtopic-content .skill-list-item a {
     color: inherit;
+    flex: 1;
+    overflow: hidden;
+    padding-right: 10px;
     text-decoration: none;
+    text-overflow: clip;
```

Comment:
Is this still relevant now that we are showing the entire content?

Reply:
Done. Removed surplus CSS.